tools/project_manager/diff.php

```diff
@@ -138,11 +141,25 @@
     echo "| <a href='$format_url'>$format_label</a>\n";
 }
 
+if (user_can_mentor_in_any_round()) {
+    // show option to change diff style
+    if ($bb_diffs) {
+        $bb_diffs_label = _("Show table-style diff");
+        $_GET["bb_diffs"] = "off";
```

Comment:
is writing to `$_GET` good practice? I assumed it was read only until now.

Reply:
I copied it from the code directly above that sets the "With/Without Formatting" flag. I don't know if it's good practice.

Reply:
It's not read only and it's not a great practice but it's better to be consistent with the other instance in this file (like you have it).